catalogs/catalog-hadoop/src/main/java/org/apache/gravitino/catalog/hadoop/HadoopCatalogOperations.java

```diff
@@ -90,6 +90,10 @@ public class HadoopCatalogOperations implements CatalogOperations, SupportsSchem
 
   private CatalogInfo catalogInfo;
 
+  private final Map<String, FileSystemProvider> fileSystemProvidersMap = Maps.newHashMap();
+
+  private String defaultFilesystemProvider;
```

Comment:
You can maintain a default `FileSystemProvider`, not just a string. Besides, it is `FileSystem`, not `FileSystem`.